docs/2_2_common_schema.adoc

```diff
@@ -906,7 +906,7 @@ The normalized string `variableName` is used to identify the terminal member var
 _[The `name` attribute of the model variable is equal to the `variableName` of the `TerminalMemberVariable`._
 _An importing tool may use the `variableName` attribute to identify a specific alias to get additional information, such as minimum, maximum, and nominal values.]_
 
-One variable can be part of several terminals, which is considered aliasing according to the rules of <<aliasing-of-variables>>.
+One variable can be part of several terminals.
```

Comment:
Is this really true for all variables? Can a input variable be part of several terminals?

Reply:
Please open a new issue if you think this is should be changed. This PR only removes the reference to alias variables.